src/components/SortButton.jsx

```diff
@@ -0,0 +1,27 @@
+import { ArrowUpIcon, ArrowDownIcon } from "@heroicons/react/20/solid";
+import {
+  QUERY_PARAM_ORDER_BY_ASC,
+  QUERY_PARAM_ORDER_BY_DESC,
+} from "../utils/constants";
+
+const getIcon = (direction, currentValue) => {
+  const cssClassName =
+    direction === currentValue
+      ? "sort-button__icon--selected"
+      : "sort-button__icon--unselected";
+
+  switch (direction) {
+    case QUERY_PARAM_ORDER_BY_ASC:
+      return <ArrowUpIcon className={cssClassName} />;
+    case QUERY_PARAM_ORDER_BY_DESC:
+      return <ArrowDownIcon className={cssClassName} />;
+  }
+};
+
```

Comment:
This is an interesting way to manage the sort by queries 😄 You can also use useSearchParams to access the query parameters directly. Here is a link for more info: [https://reactrouter.com/en/main/hooks/use-search-params](url)

Reply:
It's already using `useSearchParams`. It was implemented in previous PR #10 🙂